src/constants/game/CustomMapExperience.java

```diff
@@ -0,0 +1,21 @@
+package constants.game;
+
+import java.util.HashMap;
+
+public final class CustomMapExperience
+{
+    //Applies experience multiplier to MapID
+    public static final HashMap<Integer, Integer> CUSTOM_MAP_EXPERIENCE = new HashMap<Integer, Integer>() {{

```

Comment:
My issue with this variable is that I don't know what the two integers are.
Is the first one MapId and the other Experience? If so I would name this variable something like MapExperienceHashMap. I know the two maps sound a bit weird together